readthedocs/docsitalia/static-src/projects/css/_serp.less

```diff
@@ -1,3 +1,53 @@
+// Filters
+.serp-search-filters {
+  padding: 0;
+  margin-bottom: 35px !important;
```

Comment:
In generale, tendiamo ad usare sempre multipli di `4px` (meglio se `8px`) quando possibile, per cui `margin-bottom: 35px !important;` qui e `padding: 0px 8px;` a riga 24.

Considera anche l'uso di classi aggiuntive [già presenti su Bootstrap Italia](https://italia.github.io/bootstrap-italia/docs/organizzare-gli-spazi/spaziature/) per queste cose (come `p-1`, `pt-1`, `px-1`; nel tuo caso `px-2` per la riga 24, e così via)